.github/workflows/ci.yml

```diff
@@ -38,7 +38,9 @@ jobs:
             pkg-config `# for installing libraries with vcpkg`\
             git \
             cmake \
-            ninja-build
+            libx11-dev \
+            freeglut3-dev \
```

Comment:
Actually it doesn't work when i only install the libgl-dev, so i install libgl-dev and freeglut3-dev.

Reply:
Yes, it doesn't work because the whole of opencascade is installed without distinction, including optional features relying on freeglut. As mentioned in my previous comment, you would need to modularize the vcpkg port to avoid installing everything, or rely on CMake `FetchContent` mechanism to only build and link TKSTEP.